.github/workflows/main.yml

```diff
@@ -20,6 +20,6 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
+    - run: npm install
```

Comment:
Why the change ?

Reply:
The npm install script inside the boa-sdk-ts did not run automatically when it was 'npm ci'.